sklearn/feature_extraction/tests/test_text.py

```diff
@@ -1033,6 +1033,20 @@ def func():
     assert_raise_message(exception, message, func)
 
 
+def test_hashingvectorize_seed():
+
+    # assert same behaviour without seed
+    hv = HashingVectorizer()
+    X = hv.fit_transform(['hello world', 'hello hello', 'hello goodbye'])
```

Comment:
I would also call transform to make sure that the seed is not set in fit. You can do that with pytest

```python
@pytest.mark.parametrize("method", ['fit_transform', 'transform'])
def test_hasingvectorizer_seed(method):
    ...
    text = ['hello world', 'hello hello', 'hello goodbye']
    X = getattr(hv, method)(text)
    ...
```